default.env

```diff
@@ -122,24 +122,21 @@ HLL_HOST=
 # The HLL RCON port (this is not the game or query port), ex: 27230
 HLL_PORT=
 
-# The RCON  password of your game server
+# The RCON password of your game server
 HLL_PASSWORD=
 
 # -----------------------------
 # RCON Web Setttings
 # -----------------------------
 
-# This is the port where the Rcon Website will be accessible. Feel free to change. 
-# If you're running multiple instances you have to change it per instance
-# If you want to put a reverse proxy on your rcon (e.g. for HTTPS) and block the direct 
-# access then use: 127.0.0.1:8010
+# This is the port where the CRCON website will be accessible. Feel free to change. 
+# If you're running multiple instances, you have to change it per instance
 RCONWEB_PORT=8010
 
 # Same as above but for https
 RCONWEB_PORT_HTTPS=9010
 
 # Ports for the public version (statistics)
-# If you don't want the public site write add "127.0.0.1:" in front of the port number, example: 127.0.0.1:7010
```

Comment:
Even  though it no longer works, it still has a use case, hence I wouldn't consider this outdated. See #652

Reply:
So we should find another way to present it, as we got two or three bug reports related to this the last days (?)